crates/ruff_linter/resources/test/fixtures/flake8_pie/PIE804.py

```diff
@@ -20,3 +20,6 @@
 foo(**{"": True})
 foo(**{f"buzz__{bar}": True})
 abc(**{"for": 3})
+
+# Duplicated key names wont be fixed to avoid syntax error.
+abc(**{'a': b}, **{'a': c})  # PIE804
```

Comment:
How would this work with positional arguments? For example,
```python
def abc(a):
	pass

abc(1, **{'a': 2})
```

Correct me if I'm wrong but I guess it would fix it to `abc(1, a=2)` which isn't a syntax error although it's a runtime error.

I don't think this is in scope for this rule so maybe we could just highlight it in the docs. I'll leave this up to @zanieb as the main reviewer :)

Reply:
Yes, needs info in docs, with considering bellow code as correct:
```py
def abc(a, /, **kw): ...

abc(1, **{'a': 2})
```